src/SymfonyCache/PurgeListener.php

```diff
@@ -83,6 +96,24 @@ public function handlePurge(CacheEvent $event)
         $response = new Response();
         $store = $event->getKernel()->getStore();
 
+        // Purge whole cache
+        if ($request->headers->has($this->clearCacheHeader)) {
+            if (!$store instanceof Psr6StoreInterface) {
```

Comment:
does the default cache store of symfony not allow pruning? if not, we should mention that in the documentation

Reply:
Unfortunately, it does not. To be fair, there's very little reason not to use my store implementation 😄
How about https://github.com/FriendsOfSymfony/FOSHttpCache/pull/437/commits/aedf0771865a2df110780d447deadbb5a419757d?